### ISSUE TITLE
Docs don't match CLI

### ISSUE THREAD

**Issue by humphd** [prefix-dev/rattler-build#502]
Following the docs at https://prefix-dev.github.io/rattler-build/highlevel/#how-to-run-rattler-build

```
> rattler-build --recipe-file myrecipe/recipe.yaml

pixi run build
✨ Pixi task: rattler-build --recipe-file recipe.yaml
error: unexpected argument '--recipe-file' found

Usage: rattler-build [OPTIONS] [COMMAND]

rattler-build --version
rattler-build 0.7.0
```

**Comment by wolfv** [comment]
Fixed in #503

**Comment by wolfv** [comment]
Thanks!
